docs/UserGuide.md

```diff
@@ -491,6 +491,10 @@ Furthermore, certain edits can cause the DDD to behave in unexpected ways (e.g.,
 
 In case you need more information on the command parameters, here's a more comprehensive explanation of each parameter:
 
+- `Vendor`: A person who provides certain kinds of services
+- `Client`: A person who is getting married
+- `Event`: The wedding event
```

Comment:
This should be above the previous sentence because these are types of records and not command parameters.

Reply:
Does this mean we need to change the order of these three? Or it is just irrelevant and should be removed?